torchvision/prototype/features/_image.py

```diff
@@ -97,9 +97,10 @@ def __repr__(self, *, tensor_contents: Any = None) -> str:  # type: ignore[overr
     def new_like(
         cls, other: Image, data: Any, *, color_space: Optional[Union[ColorSpace, str]] = None, **kwargs: Any
     ) -> Image:
-        return super().new_like(
-            other, data, color_space=color_space if color_space is not None else other.color_space, **kwargs
-        )
+        # Question: Is it safe to assume data to be a tensor ?
+        out = data.as_subclass(Image)
```

Comment:
No. `features.Label(0)` is a good counter example.

Reply:
This code is inside `Image`, Label wont pass here IMO